docs/core/02-how-to-guides/01-plugin-development/03-publication/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Publication
+---
+
+## Publishing Your Plugin
+
+In the following sections, we guide you through the publication process of your plugin.
+
+After you finished the implementation and setup contract of your new plugin, you now want to publish it on the Aragon Marketplace.
```

Comment:
```suggestion
After you finished the implementation and setup contract of your new plugin, you now want to publish it within Aragon's UI.
```

Reply:
👍 Hey, would be good if you can maintain these changes if these change in the future

Reply:
I wrote:

> After you finished the implementation and setup contract of your new plugin, you now want to publish it on Aragon's repository.